tests/Test/Xrefcheck/ConfigSpec.hs

```diff
@@ -35,7 +35,7 @@ test_config =
         | flavor <- allFlavors]
   , testGroup "Filled default config matches the expected format"
     -- The config we match against can be regenerated with
-    -- stack exec xrefcheck -- dump-config -t GitHub -o tests/configs/github-config.yaml
+    -- stack exec xrefcheck -- dump-config -t GitHub -o tests/configs/github-config.yaml --force
```

Comment:
By the way, I think this should not be a fixup commit. It's fine to include in this PR, but I would form it as a standalone commit saying that it is a follow-up change after another PR (which is perfectly fine, it happens).

Reply:
Right! Actually, this change should not be here. I have already changed it in the appropriate PR.

Reply:
Ah yeah, I actually saw it there after reviewing this PR.